### PR TITLE
chore: update dev dependencies

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
No

### Motivation / Use-Case
Update dev dependencies

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: test/cli/__snapshots__/basic.test.js.snap.webpack5 [hunk 1]
```diff
@@ -33,30 +33,33 @@ exports[`basic basic should work: stderr 1`] = `
...
-  -j, --json [value]                                  Prints result as JSON or store it in a file.
-  --fail-on-warnings                                  Stop webpack-cli process with non-zero exit code on warnings from webpack
+  -j, --json [pathToJsonFile]                         Prints result as JSON or store it in a file.
+  --fail-on-warnings                                  Stop webpack-cli process with non-zero exit code on warnings from webpack.
+  -e, --extends <value...>                            Path to the configuration to be extended (only works when using webpack-cli).
   -d, --devtool <value>                               A developer tool to enhance debugging (false | eval | [inline-|hidden-|eval-][nosources-][cheap-[module-]]source-map).
   --no-devtool                                        Negative 'devtool' option.
   --entry <value...>                                  A module that is loaded upon startup. Only the last one is exported.
+  --extends <value...>                                Path to the configuration to be extended (only works when using webpack-cli).
```
Comment (snitin315) [inline test/cli/__snapshots__/basic.test.js.snap.webpack5:53]: I think some bug, we have `--extends` two times in help output.

Comment (alexander-akait) [reply test/cli/__snapshots__/basic.test.js.snap.webpack5:53]: I see, can you open an issue in webpack-cli?

Comment (snitin315) [reply test/cli/__snapshots__/basic.test.js.snap.webpack5:53]: @alexander-akait https://github.com/webpack/webpack-dev-server/pull/5030

Comment (snitin315) [reply test/cli/__snapshots__/basic.test.js.snap.webpack5:53]: Let's keep this PR for devDeps only and update after doing patch release for webpack-cli.

Moved webpack-dev-middleware uprate to separate PR -https://github.com/webpack/webpack-dev-server/pull/5032

**Comment by alexander-akait** [comment]
@snitin315 I will try to make a release today

**Comment by alexander-akait** [comment]
@snitin315 Do we need to close it or rebase?

**Comment by snitin315** [comment]
Closing this

**Pull request closed without merging**